src/components/Envelope.vue

```diff
@@ -547,6 +547,9 @@ list-item-style.draft .app-content-list-item-line-two {
 .app-content-list-item-star.icon-starred {
 	display: block;
 }
+.app-content-list-item-icon {
+	display: flex;
```

Comment:
Maybe this?

```suggestion
	display: flex;
	align-items: center;
```

Reply:
this doesnt do anything, its already added on the wrapper